device/mellanox/x86_64-mlnx_msn4410-r0/plugins/sfplpmget.py

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/plugins/sfplpmget.py
```

Comment:
are you sure? i prefer in any case it will be using the sn4700 even for visuality
please consult with Kebo if this still needed. i am not sure as we are using the new APIs. but maybe i am wrong. the same is for the other sfp scripts